values.go

```diff
@@ -333,7 +338,7 @@ func (Address) MeteredType(common.MemoryGauge) Type {
 }
 
 func (v Address) ToGoValue() any {
-	return [AddressLength]byte(v)
+	return v.String()
```

Comment:
Why change to a string here? A Go byte array seems like a better mapping

Reply:
ok